data-serving/data-service/schemas/cases.schema.json

```diff
@@ -213,110 +213,125 @@
         }
       },
       "travelHistory": {
-        "bsonType": "array",
-        "uniqueItems": true,
-        "items": {
-          "bsonType": "object",
-          "additionalProperties": false,
-          "properties": {
-            "_id": {
-              "bsonType": "objectId"
-            },
-            "location": {
+        "bsonType": "object",
+        "additionalProperties": false,
+        "properties": {
+          "_id": {
```

Comment:
Slightly orthogonal thought, but WDYT about `_id` in all of our subdocs? Does it serve any purpose if we don't plan to query the documents by ID?

Reply:
MongoDB adds this by default :\ If we want to use additionalProperties: false, we have to include it in the schema (AFAICT).

Reply:
There's [this option in Mongoose](https://mongoosejs.com/docs/guide.html#_id) to disable it. I'm not totally sure what it means by it "will refuse to save a document," but I can't imagine they'd have this for sub-docs, then prevent that parent doc from being saved. :thinking:

Reply:
many dates late but here it is! https://github.com/open-covid-data/healthmap-gdo-temp/pull/411